README.md

```diff
@@ -1,32 +1,79 @@
+
+
+## Scenario overview
+TorchDriveEnv comes bundled with a predefined collection of diverse and challenging training and testing scenarios. Training scenarios were created using the InvertedAI API, while the testing scenarios were hand tailored for interesting challenges. An overview of the testing scenarios is given in the visualization below. Note that while the ego agent has fixed waypoints in these scenarios, its starting position is sampled uniformly between the first two waypoints, and NPCs are initialized dynamically using the InvertedAI API. This results in a large variation in the available scenarios. 
+
+<table>
+  <tr>
+    <td align="center"><img src="https://github.com/inverted-ai/torchdriveenv/assets/16724505/ab82ec1e-fe79-4721-a996-512162032894.png" alt="Three Way"><br>Three Way</td>
+    <td align="center"><img src="https://github.com/inverted-ai/torchdriveenv/assets/16724505/ce50a190-065f-4f59-b010-1e503ef78696.png" alt="Chicken"><br>Chicken</td>
+    <td align="center"><img src="https://github.com/inverted-ai/torchdriveenv/assets/16724505/0ebddde4-62b0-44ad-bf40-bbb029d04589.png" alt="Parked Car"><br>Parked Car</td>
+    <td align="center"><img src="https://github.com/inverted-ai/torchdriveenv/assets/16724505/d38b72ff-f90c-4f83-8bb5-454f92168d1d.png" alt="Roundabout"><br>Roundabout</td>
+    <td align="center"><img src="https://github.com/inverted-ai/torchdriveenv/assets/16724505/1d4b8706-0bb6-4793-b57c-2b35eb020650.png" alt="Traffic Lights"><br>Traffic Lights</td>
+  </tr>
+</table>
+
+
+## Reward
+The reward for TorchDriveEnv is based on waypoints that are provided to the ego agent, as well as a reward for movement and smoothness. Episodes that cause infractions (causing collisions or going offroad) are terminated early, otherwise the episode is terminated after a fixed number of steps. For a full overview of the reward, refer to the paper linked below. 
+
 # Installation
 
-The basic installation of torchdriveenv uses an OpenCV renderer, which is slower but easy to install. PyTorch3D renderer can be faster, but it requires specific versions of CUDA and PyTorch, so it is best installed in Docker.
+The basic installation of torchdriveenv uses an OpenCV renderer, which is slower but easy to install. PyTorch3D renderer can be faster, but it requires specific versions of CUDA and PyTorch, so it is best installed using Docker.
+
+TorchDriveEnv comes with an example that integrates [Stable-Baselines3](https://stable-baselines3.readthedocs.io/en/master), which can be used for testing purposes.
 
 ## Opencv rendering
```

Comment:
`OpenCV` would be more accurate.

Reply:
Is this a comment about how to stylize it? I.e. `OpenCV` vs OpenCV?